core/src/test/java/org/opentosca/toscana/core/util/ZipUtilityTest.java

```diff
@@ -35,7 +36,7 @@ public void setUp() throws Exception {
         unzipped.mkdirs();
         original.mkdirs();
 
-        //generate folders Total 125 Directories with 10 files each (1250 files)
+        //generate folders Total 25 Directories with 10 files each (250 files)
```

Comment:
Can we reduce the file amount even more? This is not a performance test. More files add more noise and make the execution time longer. e.g. 2 folders with 2 files in each should be sufficient.

Reply:
Well reducing it does not affect execution speed in any way. At least its not really measurable